test/integration/addons_test.go

```diff
@@ -708,3 +709,16 @@ func validateGCPAuthAddon(ctx context.Context, t *testing.T, profile string) {
 		}
 	}
 }
+
+func validateHeadlampAddon(ctx context.Context, t *testing.T, profile string) {
+	defer PostMortemLogs(t, profile)
+
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "addons", "enable", "headlamp", "-p", profile, "--alsologtostderr", "-v=1"))
```

Comment:
```suggestion
        args := append([]string{"addons", "enable", "headlamp", "-p", profile, "--alsologtostderr", "-v=1"},  StartArgs()...)
	rr, err := Run(t, exec.CommandContext(ctx, Target(), args...))
```
`StartArgs` contains test environment specific flags such as `driver` and `container-runtime`

Reply:
If my understanding is correct the `StartArgs` is used when starting the minikube cluster right?